management_api_app/models/schemas/workspace_service.py

```diff
@@ -0,0 +1,59 @@
+from pydantic import BaseModel, Field
+
+from models.domain.workspace import Workspace
+
+
+def get_sample_workspace(workspace_id: str, spec_workspace_id: str = "0001") -> dict:
```

Comment:
is this supposed to be get_sample_workspace_service? and workspace_service_id?
The spec_workspace_id doesn't seem to be used